src/server/project.js

```diff
@@ -0,0 +1,346 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+var fs = require('fs'),
+    path = require('path'),
+    cordovaServe = require('cordova-serve'),
+    Q = require('q'),
+    dirs = require('./dirs'),
+    pluginUtil = require('./utils/plugins'),
+    prepareUtil = require('./utils/prepare'),
+    utils = require('./utils/jsUtils');
+
+/**
+ * @constructor
+ */
+function Project(simulator, platform) {
+    this._simulator = simulator;
```

Comment:
Cool. With telemetry and config being used together a lot, we might want to make telemetry a new config property. We don't need to do it now, but if it makes sense, we could do it in the future.